src/types.ts

```diff
@@ -126,6 +126,8 @@ export interface AdxTableSchema {
 export interface AdxColumnSchema {
   Name: string;
   CslType: string;
+  Type: string;
```

Comment:
Rather than setting many of these values to an empty string, can't we make this optional?